spec/lib/presenters/index_presenter_spec.rb

```diff
@@ -4,6 +4,7 @@
 describe IndexPresenter do
   describe "#edit_path" do
     it "returns the edit path for a resource" do
+      raise 'ohno'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.